Add Flipnote Lenny exploit setup page for users without Unlaunch (#139)
cfw-guide/dsi.cfw.guide: docs/launching-the-flipnote-exploit.md

---
title: Launching the Exploit (Flipnote Lenny)
---

We'll start by first setting up your SD card to launch our exploit. For the best Unlaunch-free experience, we recommend using an exploit called "Flipnote Lenny" which takes advantage of a flaw in how the Flipnote Studio application handles flipnotes.

Launching the exploit will boot you into TWiLight Menu++, a homebrew application that acts as a replacement to the DSi Menu.

## Requirements
- A way to copy downloaded files to your SD card
- An application that can extract archives, such as [7-Zip](https://www.7-zip.org/) (Windows) or [The Unarchiver](https://apps.apple.com/us/app/the-unarchiver/id425424353) (macOS)
   - We advise you to not use WinRAR, as it is known to break things
- Flipnote Studio installed on your Nintendo DSi

>    - If you do not have Flipnote Studio, you may still be able to redownload it from the Nintendo DSi Shop, if you have downloaded it previously
>    - If you are unable to redownload it, then go to the previous page for Memory Pit instead

Reviewer: ```suggestion
   - If you do not have Flipnote Studio, then go to the previous page and use Memory Pit instead
```
The shop is dead, no use checking it.
